src/main/algorithms/sort/insertionSort.ts

```diff
@@ -0,0 +1,51 @@
+import { ProtocolBuilder } from '@/main/simulation/protocolBuilder'
+import type { HighlightedIndex, SortAlgorithmImplementation, SortSimulation, SortSimulationStep } from '@/main/algorithms/sort/types'
+import type { TrackableProgress } from '@/main/progressTracker/types'
+
+export class InsertionSort implements SortAlgorithmImplementation {
+    sort(values: number[], progressTracker?: TrackableProgress): SortSimulation {
+        progressTracker?.init(values.length * ((values.length - 1) / 2))
+        const pB = new ProtocolBuilder<SortSimulationStep>()
+        pB.step({
+            sortedValues: values,
+            highlightedIndices: [],
+        })
+        let item = -1
+        for (let currentElement = 1; currentElement < values.length; currentElement++) {
+            for (let pointer = currentElement - 1; pointer >= 0; pointer--) {
+                if (values[pointer] > values[pointer + 1]) {
```

Comment:
Invert this if please.
This should result in the following structure:
if (negated ...) {
  ...
  break;
}
code that was in the if block
...